Fix error in Teltonika GPS APN configuration command (#55)
EMnify/product-docs: docs/quickstart/devices/gps-trackers.md

---
# GPS trackers

:::info

> For GPS vendors that aren't listed, please consult the respective manual and configure the APN to be `em` or `emnify`.

Reviewer: ```suggestion
For GPS vendors that aren't listed, please consult their respective manual and configure the APN to be `em` or `emnify`.
```

The two leading spaces are required.
:::

:::info
Refer to the [Teltonika FMB Device Family Parameter list](https://wiki.teltonika-gps.com/view/Template:FMB_Device_Family_Parameter_list) for all available parameters.
:::

## Ruptela GPS APN configuration

*Applies to HCV5, LCV5, Pro5, Trace5/NA, FM-Tco4 HCV/HCV 3G, FM-Tco4 LCV/LCV 3G, FM-Pro4/Pro4 3G, FM-Eco4/4+, FM-Eco4 light/light+/3G, FM-Eco4 S Series, FM-Eco4 T Series, FM-Plug4*

[Source Ruptela Documentation](https://doc.ruptela.lt/display/AB/Tracking+devices)

> Configuring the APN for Ruptela GPS trackers can be done through:

Reviewer: I'm not sure I why I didn't notice this earlier. 

We should either strike **through** or **Via** because both cannot be read together without it sounding wrong.

Configuring the APN for Ruptela GPS trackers can be done through: 

1. The SMS console of the [emnify Portal](https://portal.emnify.com/)

Also, note the change in the 2nd enumerated item above.

Author: Great points! These should definitely be considered when we verify and update the content for these pages (maybe you can add a comment to that ticket).

[SMSpassword] setconnection em
```

The \[SMSpassword\] can be set up in the Ruptela device center.

> If it isn't set, you can omit the SMSpassword, and the command is:

Reviewer: The current construction with a compound sentence using "and" as the conjunction makes it sound like one is trying to maximize the suspense of the information that is about to be announced, e.g., " ... _and_ the winning (lottery) number is ....!" 

```suggestion
If the SMSpassword isn't set, the command is:
```
Also, we might want to investigate the orthography of "SMSpassword" given the way it is written on lines 155, 165, and 219.

Author: These were part of your original Quickstart PR, so my only goal here was to clean up some of the formatting that was missed 🤷🏼‍♀️  I'll commit your suggestions then it'll be ready for re-review!

APN em#
```

> For some Concox models (e.g., TR02), the password needs to be sent with the command:

Reviewer: This might read better.
```suggestion
For some Concox models, for instance TR02, the password needs to be sent with the command:
```
